src/ts/common/entities/config.ts

```diff
@@ -13,20 +13,23 @@ enum ConfigKey {
 	SizeData = "size-data",
 	LatestRender = "last-render",
 	FormData = "form-data",
+	EnforceAuthAndTagIndex = "enforce-auth-tag-index",
```

Comment:
this feels like a bad name but idk what else to call it

Reply:
This is fine but we could do `EnforceTagIndexAccess`? Dunno if that's better